packages/Telephony/Actions/BatchTerminationCharacterInput.cs

```diff
@@ -68,5 +80,28 @@ public override Task<DialogTurnResult> ContinueDialogAsync(DialogContext dc, Can
                 return Task.FromResult(new DialogTurnResult(DialogTurnStatus.Waiting));
             }
         }
+
+        /// <inheritdoc/>
+        protected override async Task<bool> OnPreBubbleEventAsync(DialogContext dc, DialogEvent e, CancellationToken cancellationToken)
```

Comment:
Question: can we have OnPreBubbleEvent and AllowInterruption properties be in the base class so that we have that code only once?

Reply:
Done